sass/_layout.scss

```diff
@@ -0,0 +1,76 @@
+.container {
+    background-color: $color-divider;
+    width: 100vw;
+    height: 900px; // later will be defined by the content
+    padding: 2rem;
+    
+    display: grid;
+    grid-gap: .8rem;
+    
+    grid-template-rows: .07fr .86fr .07fr;
```

Comment:
.07fr .86fr .07fr; - it's a pain )

if you want to use fr, you could just write: 1fr 12fr 1fr. You just need to keep proportions. 
If you want the fixed size, use "rem": 6rem 1fr 6rem; In this case you will have fixed size header and footer, but flexible content